api/src/main/java/uk/gov/hmcts/payment/api/controllers/PaymentGroupController.java

```diff
@@ -332,6 +361,180 @@ private Payment getPayment(PaymentFeeLink paymentFeeLink, String paymentReferenc
     }
 
+    @ApiOperation(value = "Record a Bulk Scan Payment", notes = "Record a Bulk Scan Payment")
+    @ApiResponses(value = {
+        @ApiResponse(code = 201, message = "Bulk Scan Payment created"),
+        @ApiResponse(code = 400, message = "Bulk Scan Payment creation failed"),
+        @ApiResponse(code = 422, message = "Invalid or missing attribute")
+    })
+    @PostMapping(value = "/payment-groups/{payment-group-reference}/bulk-scan-payments-strategic")
+    @ResponseBody
+    @Transactional
+    public ResponseEntity<PaymentDto> recordBulkScanPaymentStrategic(@PathVariable("payment-group-reference") String paymentGroupReference,
+                                                                     @Valid @RequestBody BulkScanPaymentRequestStrategic bulkScanPaymentRequestStrategic,
+                                                                     @RequestHeader(required = false) MultiValueMap<String, String> headers) throws CheckDigitException {
+        // Check Any Duplicate payments for current DCN
+        if (bulkScanPaymentRequestStrategic.getDocumentControlNumber() != null) {
+            List<Payment> existingPaymentForDCNList = payment2Repository.findByDocumentControlNumber(bulkScanPaymentRequestStrategic.getDocumentControlNumber()).orElse(null);
+            if (existingPaymentForDCNList != null && !existingPaymentForDCNList.isEmpty()) {
+                throw new DuplicatePaymentException("Bulk scan payment already exists for DCN = " + bulkScanPaymentRequestStrategic.getDocumentControlNumber());
+            }
+        }
+
+        //Bulk scan payments call
+        List<SiteDTO> sites = referenceDataService.getSiteIDs();
+
+        if (sites.stream().noneMatch(o -> o.getSiteID().equals(bulkScanPaymentRequestStrategic.getSiteId()))) {
+            throw new PaymentException("Invalid siteID: " + bulkScanPaymentRequestStrategic.getSiteId());
+        }
+
+        PaymentProvider paymentProvider = bulkScanPaymentRequestStrategic.getExternalProvider() != null ?
+            paymentProviderRepository.findByNameOrThrow(bulkScanPaymentRequestStrategic.getExternalProvider())
+            : null;
+
+        Payment payment = Payment.paymentWith()
+            .reference(referenceUtil.getNext("RC"))
+            .amount(bulkScanPaymentRequestStrategic.getAmount())
+            .caseReference(bulkScanPaymentRequestStrategic.getExceptionRecord())
+            .ccdCaseNumber(bulkScanPaymentRequestStrategic.getCcdCaseNumber())
+            .currency(bulkScanPaymentRequestStrategic.getCurrency().getCode())
+            .paymentProvider(paymentProvider)
+            .serviceType(bulkScanPaymentRequestStrategic.getService().getName())
+            .paymentMethod(PaymentMethod.
+                paymentMethodWith().name(bulkScanPaymentRequestStrategic.getPaymentMethod().getType())
+                .description("Cheque")
```

Comment:
Why are we trying to persist all the Bulk Scan Payments as Cheque, they could be [Cash/Cheque/Postal Order]

Reply:
This need changes